server/api/routes-auth.js

```diff
@@ -192,26 +194,6 @@ router.get('/auth/whois', async (ctx) => {
   ctx.body = body
 })
 
-/**
- * This is to compensate using localStorage for token
- * Ideally, this should NOT be used as-is for a production Web App.
- * Only moment you’d want to expose token is if you have SysAdmins
- * who wants to poke APIs manually and needs their JWT tokens.
- */
-router.get('/auth/token', async (ctx) => {
-  ctx.assert(ctx.session.jwt, 401, 'Requires authentication')
-  let body = {}
-  try {
-    const token = ctx.session.jwt
-    body.jwt = token
-  } catch (e) {
-    // Nothing to do, body.authenticated defaults to false. Which would be what we want.
-  }
-
-  ctx.status = 200
-  ctx.body = body
-})
-
```

Comment:
We don't need this anymore.
The only time we should receive token is when we got a response from the authentication.